.travis.yml

```diff
@@ -1,20 +1,23 @@
 language: android
 jdk: oraclejdk8
 sudo: required
+
 env:
   global:
   - GRADLE_OPTS="-Xmx512m"
   matrix:
- # - ADB_INSTALL_TIMEOUT=15 ANDROID_BUILD_TOOLS_VERSION=24.0.2 ANDROID_TARGET=android-23 ANDROID_ABI=armeabi-v7a
-  - ADB_INSTALL_TIMEOUT=15 ANDROID_BUILD_TOOLS_VERSION=23.0.2 ANDROID_API_LEVEL=23 ANDROID_ABI=armeabi-v7a ANDROID_TAG=google_apis
+  - ADB_INSTALL_TIMEOUT=15 ANDROID_BUILD_TOOLS_VERSION=25.0.3 ANDROID_TARGET=android-25 ANDROID_ABI=armeabi-v7a
+
 cache:
   directories:
   - "${TRAVIS_BUILD_DIR}/android/gradle/caches/"
   - "${TRAVIS_BUILD_DIR}/android/gradle/wrapper/dists/"
   - "$HOME/android/.gradle/caches/"
   - "$HOME/android/.gradle/wrapper/"
+
 android:
   components:
+  - tools
```

Comment:
`-tools` is already present. Any specific reasons to add it again? In case you did it by mistake please rectify it.